src/modules/tests/test-data/common/eyesight-test/eyesight-test.selector.ts

```diff
@@ -0,0 +1,4 @@
+import { EyesightTest } from '@dvsa/mes-test-schema/categories/common';
```

Comment:
left over from barrel changes i think. can we revert please.

Reply:
This is from the schema, no barrel file